### PR TITLE
feat: add cooldown phase modeling to firing curves

### PR DESCRIPTION
**Pull request by bo0tzz**
## Summary
- Implements Newton's Law of Cooling to model the natural cooldown phase after firing
- Adds kiln preset system with calibrated cooling coefficients
- Provides both simple and advanced configuration modes

### Core Functionality
- **Implicit cooldown**: Automatically added after firing segments, no manual segment needed
- **Newton's Law of Cooling**: Scientifically accurate exponential decay model: `T(t) = T_ambient + (T_initial - T_ambient) * e^(-kt)`
- **Real-world calibration**: Based on actual kiln data (25L, 75mm soft brick, k ≈ 0.26)

### Configuration
- **Simple mode**: Select from preset kiln types and cooling speeds (slow/normal/fast)
- **Advanced mode**: Direct control of cooling coefficient (k), ambient temp, and stop temp
- **Custom kiln types**: Save your own kiln configurations for reuse
- **Persistent settings**: All cooldown settings saved to localStorage

### Visual Design
- **Clear distinction**: Thinner blue line (2px) for cooldown vs thick line (4px) for firing
- **Transition marker**: "End of Firing" label at the exact transition point
- **Clean presentation**: Removed data point circles from cooldown phase

### Kiln Presets
Includes 9 calibrated kiln types based on size and insulation:
- Tiny/Test Kiln (< 25L)
- Small Hobby kilns (25-50L) with various insulation types
- Medium Hobby (50-100L)
- Studio kilns (100-400L)
- Specialty kilns (Gas Reduction, Raku)

## Test Plan
- [x] Verify cooldown curve appears after firing segments
- [x] Test simple mode kiln preset selection
- [x] Test advanced mode k-value adjustment
- [x] Save and load custom kiln types
- [x] Verify settings persist across page refresh
- [x] Test edge cases (very low/high k values)
- [x] Confirm visual distinction between phases

🤖 Generated with [Claude Code](https://claude.ai/code)

### CODE REVIEW

File: src/lib/components/SettingsModal.svelte [hunk 1]
```diff
@@ -52,6 +96,34 @@
 		editingProfileName = '';
 		editingProfileDescription = '';
 	}
+
+	function saveCustomKiln() {
+		if (customKilnName.trim()) {
+			// Create new kiln
+			const newKiln = {
+				id: `custom-${Date.now()}`,
```
Comment (bo0tzz) [inline src/lib/components/SettingsModal.svelte]: Won't this cause conflicts?

File: src/lib/components/SettingsModal.svelte [hunk 2]
```diff
@@ -52,6 +96,34 @@
...
+				description: customKilnDescription.trim(),
+				defaultK: cooldownSettings.coolingCoefficient || 0.15,
+				kRange: [(cooldownSettings.coolingCoefficient || 0.15) * 0.8, (cooldownSettings.coolingCoefficient || 0.15) * 1.2]
+			};
+			
+			// Update local state first
+			customKilns = [...customKilns, newKiln];
+			
+			// Save to localStorage
+			localStorage.setItem('customKilns', JSON.stringify(customKilns));
```
Comment (bo0tzz) [inline src/lib/components/SettingsModal.svelte]: Stuff like this should all be handled centrally in storage.ts so it's nicely abstracted.
